collections/_article/gdc-2023-godot-games.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "Godot & Friends at GDC 2023"
+excerpt: "We are going to be at the Games Developer Conference (GDC) 2023, which will take place in San Francisco from March 20th to March 24th."
+categories: ["events"]
+author: Emi
+image: /storage/blog/covers/gdc2023.jpg
+date: 2023-03-06 09:00:00
+future: true
+---
+
+<style>
+	article .row-2 {
+		display: grid;
+		gap: 10px;
+	}
+	@media (min-width: 768px) {
+		article .row-2 {
+			grid-template-columns: 1fr 1fr;
+			gap: 30px;
+		}
+	}
+	article .row-2 h3 {
+		margin-top: 10px;
+		margin-bottom: -5px;
+	}
+	article .media {
+		display: grid;
+	}
+	article .media a.lightbox {
+		pointer-events: none;
+	}
+	article .media img {
+		max-width: 100%;
+		border-radius: 7px;
+		background-color: var(--card-background-color);
+		box-shadow: 0 5px 10px -3px #00000078;
+		display: block;
+		margin-bottom: 6px;
+	}
+</style>
+
+
+We are going to be at the Games Developer Conference (GDC) 2023, which will take place in San Francisco from March 20th to March 24th.
+
+At GDC 2023, we will have our own booth where we will be hosting game developers who will be showcasing their latest games.
```

Comment:
```suggestion
For the first time, Godot is going to have its own booth at GDC where we will be hosting game developers who will be showcasing their latest games made with the engine.
```

Reply:
Though this sentence is just weird, as it kind of repeats both preceding and following sentiments.